lusidtools/cocoon/config/seed_sample_data.json

```diff
@@ -0,0 +1,36 @@
+{"instruments": {
+    "identifier_mapping": {
+        "ClientInternal": "txn_instrument_id",
```

Comment:
I think this should just be `instrument_id`

Reply:
Done.